Sources/Widgets/Widgets3D/ResliceCursorWidget/example/index.js

```diff
@@ -182,11 +182,19 @@ reader.setUrl(`${__BASE_PATH__}/data/volume/LIDC2.vti`).then(() => {
         // No need to update plane nor refresh when interaction
         // is on current view. Plane can't be changed with interaction on current
         // view. Refreshs happen automatically with `animation`.
-        .filter((_, index) => index !== i)
+        // Note: Need to refresh also the current view because of adding the mouse wheel
+        // to change slicer
+        // .filter((_, index) => index !== i)
```

Comment:
this will dramatically reduce performances. 

I'm surprised the view does not refresh itself when scrolling.
Alternatively, pass a parameter to the InteractionEvent to know if you need to refresh self

Reply:
Thanks to the previous modifications (remove the render on renderwindow) then it make sens to keep it has it is now.
If you translate/rotate on view X, it will only update the moving views. If you scroll on view X, it will only update the view X thanks to `modified` attribute in `updateReslice` function

Reply:
If you scroll on the view X, you need to refresh the views Y and Z because the line X has changed

Reply:
The updateReslice function only modify the Reslice filters. Not the representation of the reslice cursor (spheres and lines).
The representation is updated thanks to the requestAnimation. That's what I understand of the request Animation